generators/Program.fs

```diff
@@ -48,21 +36,32 @@ let private regenerateTestClasses options =
     let regenerateTestClass' = regenerateTestClass options
     
     createExercises options
-    |> List.filter (shouldBeIncluded options)
+    |> List.filter (isNotFilteredByName options)
     |> function
-        | [] -> Log.Warning"No exercises matched given options."
+        | [] -> Log.Warning "No exercises matched given options."
         | exercises ->
             List.iter regenerateTestClass' exercises
             Log.Information("Re-generated test classes.")
+            
+            
+                      
 
 [<EntryPoint>]
 let main argv = 
     Logging.setupLogger()
 
     match parseOptions argv with
+    | Ok(options) when options.Status.IsSome && options.Exercise.IsSome -> 
+        Log.Error("Can't have both -s and -e.")
```

Comment:
Maybe reword this slightly, as the user could also have specified the arguments as `--exercise` and `--status`?

Reply:
Sounds good!